src/renderer/chunk_meshing.zig

```diff
@@ -1268,16 +1272,18 @@ pub const ChunkMesh = struct { // MARK: ChunkMesh
 			self.lastNeighborsSameLod[chunk.Neighbor.dirUp.toInt()] = null;
 		}
 		self.mutex.unlock();
-		self.generateMesh(&lightRefreshList); // TODO: Batch mesh updates instead of applying them for each block changes.
-		self.mutex.lock();
-		for(lightRefreshList.items) |other| {
-			if(other.needsLightRefresh.load(.unordered)) {
-				other.scheduleLightRefreshAndDecreaseRefCount1();
-			} else {
-				other.decreaseRefCount();
+
+		appendIfNotContainedAndIncreaseRefCount(regenerateMeshList, self);
```

Comment:
The naming is confusing. Function that contain increaseRefCount are a sign to the caller to decrease it again, which does not happen here.

Reply:
So remove that part completely?

Reply:
yes